docs/how-to/allow-ip-addresses.mdx

```diff
@@ -0,0 +1,35 @@
+# Allow IP Addresses 
+
+:::info 
+This guide is recommended for users bring-your-own-cloud(BYOC) and using our managed cloud. Zephyr Cloud leverages many other IP ranges for various product and services but some of these address will not make connections to your origin. 
```

Comment:
`bringing-your-own-cloud` - idk if this needs reworded in general. 

This guide is recommended for users who want to "bring your own cloud" (BYOC) or using our managed cloud.

Reply:
I agree with @swalker326

Reply:
https://github.com/ZephyrCloudIO/zephyr-documentation/pull/27/commits/1306d2569bd71baff915d22883f40f719909844f